drivers/flash/flash_hp_ra_ex_op.c

```diff
@@ -0,0 +1,379 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sys/types.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/flash/ra_flash_api_extensions.h>
+#include <zephyr/kernel.h>
+
+#ifdef CONFIG_USERSPACE
+#include <zephyr/syscall.h>
+#include <zephyr/syscall_handler.h>
+#endif
+
+#include <soc.h>
+#include "flash_hp_ra.h"
+
+#define FLASH_HP_CONFIG_SET_ACCESS_WORD_CNT (8U)
+
+#if (FLASH_HP_CFG_CODE_FLASH_PROGRAMMING_ENABLE == 1)
```

Comment:
Please use the macro that is defined in zephyr in this context.

Reply:
I updated for this case and similar cases too. Thank you.